examples/python/custom_processing.py

```diff
@@ -135,12 +140,21 @@ def main():
     response = client.modelInfer(worker_name, request)
     assert not response.isError(), response.getError()
     outputs = response.getOutputs()
+    i=0;
     for output in outputs:
         assert output.datatype == proteus.DataType.INT8
         recv_data = output.getInt8Data()
         # Can optionally post-process the result
         k = postprocess(recv_data, 5)
-        assert k == gold_response_output
+        print("Input filename: ", image_paths[i])
+        print("classification: ", k)
+        print("1: ",linecache.getline("../../tests/assets/imagenet_simple_labels.json", k[0]+2))
```

Comment:
Instead of linecache and using fixed offsets which are brittle, use the json module to read the file and then index into the array directly. Also use `root` instead of the `../..` when opening the file